### ISSUE TITLE
Failed model uploads are not being updated.

### ISSUE THREAD

**Issue by carmenfan** [3drepo/3drepo.io#548]
I uploaded a file which failed. the bouncer processed it and returned an error code. But the front end was not updated. 

Bouncer log shows that it got the work, sent the processing signal, then returned an error message:
![image](https://user-images.githubusercontent.com/11945337/31941697-9ec7f484-b8ba-11e7-8a09-7282a4017693.png)

But on the front end it still says processing:
![image](https://user-images.githubusercontent.com/11945337/31941717-af19e4f0-b8ba-11e7-898f-c49ed0542037.png)

Looking at the repo.io log, it has received the error, but it looks like it didn't change the status...? 
![image](https://user-images.githubusercontent.com/11945337/31941936-81081d42-b8bb-11e7-8dc3-896983cec1ed.png)

Upon refreshing the model still thinks its processing, which suggests that failures are not being written to the database. It looks like failed processing is being overlooked. There's also some overlooked reject warnings on the log which we should really look into.

Success seems to return fine. 

I might look into this tomorrow if I've done everything else.

**Comment by carmenfan** [comment]
@Charence  I've fixed this now. basically this was related to the rejected promise exception on the log. In the importFail() function you were interacting with an undefined variable `err`, which looks like you were expecting should be where the error information comes from, but as it's not passed in it doesn't exist anywhere so it was breaking the function. (see [here](https://github.com/3drepo/3drepo.io/commit/d3361c25bac805c3469a26a7f8b22a41ffedc6ef#diff-d29437b0fdc44aecb506dd86c818a84eR130))

**Comment by carmenfan** [comment]
I've added a temp fix re:  unhandled promise exception. This will need to be worked on properly on #459 

Also I've changed the logic slightly on the queue consuming function (see [here](https://github.com/3drepo/3drepo.io/commit/bfa48d19e27a5a90c6369a30bdf9e5063661f408#diff-0d946539f6011af83aaee001fc817df0R359)) So it'll processing the success/failure even when there are no promise. As that's the whole point of the change, being able to process the work from the queue even when we've restarted the server.
